CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Contributions
 
 We expect all contributions to conform to our
-[style guide](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/styleguide.rst),
+[style guide]( https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/styleguide.md),
```

Comment:
Would you remove the space before `http:`?

Reply:
thank you